index.html

```diff
@@ -176,7 +176,7 @@ <h2 class="work-title hidden-on-small left">Uber Navigation</h2>
       <div class="big-about">
         <div class="left-part">
           <h2 class="title myself">About<br>Myself</h2>
```

Comment:
- [ ] You have done a great job making your portfolio more accessible but here is an issue with "Text headings", remember that heading (`<hX>`) levels should have a meaningful hierarchy and you are using an `<h2>` as a section title, but you are also using the same `<h2>` tag for your project titles. I think it could be a good idea to keep the `<h2>` tag for the sections titles and give a "less" important tag (semantically speaking) to the project titles :wink:

  ![image](https://user-images.githubusercontent.com/81522435/139878695-4c145348-5ac6-4eed-8cb9-d4e8ba809734.png)

Reply:
Hi, @alonsofl! I think this is a kind of misunderstanding of the concept I am using in the design. On the lines 147-148 I have:
`<h2 class="work-title hidden-on-big">Multi-Post Stories</h2>`
`<h2 class="work-title hidden-on-small left">Uber Navigation</h2>`
These are not nested <hx>'s. These are one and the same `<h2>` headings shown on different screen sizes. There is nothing semantically wrong here. When you compare it to the next section "About" on line 178, it is again `<h2>` because it is at the same level, but in a different section. The "Uber Navigation" is part of the "Works" section and has only `<h2>` levels of headings. The "About" is a different section on the same level as the "Works" section, so the heading will remain the same. Please, let me know if you disagree. I am sending a snapshot from the Dev Tools plugin report on the structure of my website:

![image](https://user-images.githubusercontent.com/8495547/139919680-18b5e798-30a3-4090-b4aa-81ccfdcd78f1.png)

I have only `<h1>` and `<h2>` tags in the HTML file, but some of them are shown on a small screen, some of them shown on a big screen, as per Figma design. Please, have a look:

https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848

So, I will not change the headings.

Reply:
Hello @Hombre2014, I get your point about the headings and I noticed that you use different titles for different breakpoints. My suggestion is because headings `<hX>` give different "importance" to a title (semantically speaking). That's why I recommend using the `<h2>` tag for the Section title, which I think is more important than each project title (consider those as title levels). For example, if you see the other templates, these have a project section title, so this could be the same case and each project title would use the `<h3>` tag because these are "inside" or have a "lower-level" of importance. Once again, headings are not for styling/text size (for that we could use CSS), headings represent the importance of a title. I hope this explanation is a bit clearer :raised_hands:

Reply:
Hi @alonsofl. I also see your point and when I looked at the other templates, yes, there are "My recent works" or "My projects" titles and then a list of the recent project. In this case, of course, it is semantically correct to use the `<h2>` title for the Projects and the `<h3>` titles for each project itself. This is valid for all of templates 2, 3, 4 and 5. But there is no such title or heading on template 1! Please, have a look. That is why I do not have h2 for My Recent Projects because I do not have a section title My recent project. I only have 4 projects listed! Template number one is without this type of title. If you want it, please, correct the template!

Reply:
@ Hombre2014, I gave that as an example, I am aware that your template does not have a title for the Projects section. But with that premise, isn't the "About Section title" more important than each project title? Also, is this "About Section title" less important than your headline title? Then why would you use the `<h2>` for a section title and also for a less important title? Or do you consider that the "About section title" is equally as important as each project?

Reply:
@alonsofl Well, we can continue arguing till the end of ... I see your point that About is the same level as My Recent Works and they deserve to have an h2 titles, but the template has not My Recent works section title, as I already said. I either have to introduce a new h2 title My Recent works or Projects as in all other templates and then each project will have an h3 title, but by doing this I will not follow the template on Figma. So, please, either fix the template or accept my point of view. On Figma they specify the requirements as follow:

Header/2:

![image](https://user-images.githubusercontent.com/8495547/139945658-d805fcb4-c611-43cc-8cfb-92ca79c55721.png)

Header/2:

![image](https://user-images.githubusercontent.com/8495547/139945734-487df415-a9b6-465a-b71e-dcd08f6f2958.png)